src/main/java/com/conveyal/datatools/manager/controllers/api/ServerController.java

```diff
@@ -328,6 +331,53 @@ private static void validateFields(Request req, OtpServer server) throws HaltExc
         }
     }
 
+    /**
+     * Asynchrnously validates all ec2 config of a particular OtpServer instance.
```

Comment:
In my experience, this has resulted in faster validation times because everything is now happening asynchronously. Previously, each validation check would run synchronously which meant that each validation check needed to wait for the previous one to finish and since there are a lot of calls to various AWS services this would take a while. The amount of time it took to setup a thread pool was much less than the time to make each validation check synchronously.